error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/ExplicitArgumentEnumerationTest.java

```diff
@@ -24,6 +27,9 @@ void identification() {
             "  // BUG: Diagnostic contains:",
             "  private final int value = unaryMethod(ImmutableList.of(1, 2));",
             "",
+            "  private final Builder value2 =",
+            "      Counter.builder(\"foo\").tags(ImmutableList.of(Tag.of(\"bar\", \"baz\")));",
```

Comment:
The existing field declaration is there to cover a branch in the `isLocalOverload`; we can simplify this test case by moving the expression to the method below, and omitting the field declaration.